_pages/production.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Production deployment
+lead: >
+  Once you’ve tested your integration in our sandbox environment, you can request deployment to the Login.gov production environment.
+redirect_from:
+  - /production-deployment/
+sidenav:
+  - text: Production endpoints
+    href: "#production-endpoints"
+  - text: Deployments
+    href: "#deployments"
+  - text: Confirm IAA
+    href: "#confirm-iaa"
+  - text: Staging environment
+    href: "#staging-environment"
+  - text: Production configuration
+    href: "#production-configuration"
+  - text: Request deployment
+    href: "#request-deployment"
+  - text: Deploying changes
+    href: "#deploying-changes"
+---
+
+
+## Production endpoints
+
+Our integration documentation includes endpoint urls for our sandbox environment https://idp.int.identitysandbox.gov/. Our production environment is located at https://secure.login.gov/. The URL path to each endpoint remains the same. Only the domain will change. For example, the authorization endpoint will change as follows:
+* OpenID Connect - [https://secure.login.gov/openid_connect/authorize](https://secure.login.gov/openid_connect/authorize)
+* SAML - [https://secure.login.gov/api/saml/auth2020](https://secure.login.gov/api/saml/auth2020)
+
+Please be aware that the IDP certificate (X509Certificate) in the Production environment is different from the IDP certificate in the sandbox environment. The Production IDP certificates can be found here:
+* OpenID Connect - [https://secure.login.gov/api/openid_connect/certs](https://secure.login.gov/api/openid_connect/certs)
+* SAML - [https://secure.login.gov/api/saml/metadata2020](https://secure.login.gov/api/saml/metadata2020)
```

Comment:
two notes:
1. In Kramdown, we can use `<>` syntax when a URL and its text are the same (easier maintenance of the page)
2. What do you think of colons as separators instead here?
```suggestion
* OpenID Connect: <https://secure.login.gov/openid_connect/authorize>
* SAML: <https://secure.login.gov/api/saml/auth2020>

Please be aware that the IDP certificate (X509Certificate) in the Production environment is different from the IDP certificate in the sandbox environment. The Production IDP certificates can be found here:
* OpenID Connect: <https://secure.login.gov/api/openid_connect/certs>
* SAML: <https://secure.login.gov/api/saml/metadata2020>
```